Source/WebCore/rendering/cocoa/RenderThemeCocoa.mm

```diff
@@ -75,6 +76,25 @@
 #import <pal/ios/UIKitSoftLink.h>
 #endif
 
+#if USE(APPLE_INTERNAL_SDK)
+#import <WebKitAdditions/RenderThemeCocoaAdditionsBefore.mm>
+#else
+
+constexpr auto logicalSwitchHeight = 31.f;
+constexpr auto logicalSwitchWidth = 51.f;
+
+static bool renderThemePaintSwitchThumb(OptionSet<ControlStyle::State>, const RenderObject&, const PaintInfo&, const FloatRect&, const Color&)
```

Comment:
(Also `WebCore::` namespacing in this OpenSource fallback codepath)

r=me with build fixes.

Reply:
Thanks!